src/components/ui/toast.tsx

```diff
@@ -28,6 +28,7 @@ const toastVariants = cva(
       variant: {
         default: 'border bg-background text-foreground',
         destructive: 'destructive group border-destructive bg-destructive text-destructive-foreground',
+        success: 'bg-emerald-500 text-white',
```

Comment:
```css
--success-foreground: ;
--success: ;
```

Update in `globals.css`, `tailwind.config.ts` and `bg-success text-success-foreground`